package.json

```
@@ -26,7 +26,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@edx/brand": "npm:@edx/brand-edx.org@^2.0.3",
+    "@edx/brand": "npm:@edx/brand-openedx@^1.2.0",
```

Do you mind making the following change?

```suggestion
    "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
```

We're moving the package from `edx` to `openedx`, as per https://github.com/openedx/axim-engineering/issues/23.